docs/docs-beta/docs/guides/schedules.md

```diff
@@ -17,49 +19,46 @@ Schedules enable automated execution of jobs at specified intervals. These inter
 
 A basic schedule is defined by a `JobDefinition` and a `cron_schedule` using the `ScheduleDefinition` class. A job can be thought of as a selection of assets or operations executed together.
 
-<CodeExample filePath="guides/automation/simple-schedule-example.py" language="python" title="Simple Schedule Example" />
+<CodeExample filePath="guides/automation/simple-schedule-example.py" language="python" />
 
 ## Run schedules in a different timezone
 
-By default, schedules without a timezone will run in Coordinated Universal Time (UTC). If you want to run a schedule in a different timezone, you can set the `timezone` parameter.
+By default, schedules without a timezone will run in Coordinated Universal Time (UTC). To run a schedule in a different timezone, set the `timezone` parameter:
 
 ```python
 daily_schedule = ScheduleDefinition(
     job=daily_refresh_job,
     cron_schedule="0 0 * * *",
+    # highlight-next-line
     timezone="America/Los_Angeles",
 )
 ```
 
-## Run schedules on a partitioned asset
+## Create schedules from partitions
 
-If you have a partitioned asset and job, you can create a schedule using the partition with `build_schedule_from_partitioned_job`.
-The schedule will execute as the same cadence specified by the partition definition.
+If using partitions and jobs, you can create a schedule using the partition with `build_schedule_from_partitioned_job`. The schedule will execute at the same cadence specified by the partition definition.
 
-<CodeExample filePath="guides/automation/schedule-with-partition.py" language="python" title="Schedule with partition" />
+<Tabs>
```

Comment:
I'm not sure about these tabs, but I split things up like this because I found the original copy to be a little redundant / confusing.